test/unittests/common_query/test_common_query.py

```diff
@@ -25,12 +25,17 @@ def test_common_query_events(self):
         self.bus.emitted_msgs = []
         self.assertEqual(self.cc.skill_id, "common_query.openvoiceos")
 
-        qq_ctxt = {"source": "audio", "destination": "skills", 'skill_id': self.cc.skill_id}
```

Comment:
can we turn this into a end2end test?

Reply:
Probably.. I mostly wanted to make sure any changes in Messages aren't breaking